src/main/java/dk/cngroup/lentils/repository/StatusRepository.java

```diff
@@ -16,6 +16,7 @@
     Status findByTeamAndCypher(Team team, Cypher cypher);
     List<Status> findByTeam(Team team);
     Boolean existsStatusByCypherAndTeam(Cypher cypher, Team team);
+    boolean existsStatusByCypherAndTeamAndCypherStatusEquals(Cypher cypher, Team team, CypherStatus cypherStatus);
```

Comment:
Je tady nutne to `...Equals`?

Reply:
asi neni, asi to bude fungovat stejne i bez toho..